# Dalmira Hubrich
## Computational Biologist - Data Scientist
+49 157 34517760 / dalmiramer@gmail.com / Mainz, Germany / LinkedIn / GitHub

## Profile

As a computational biologist and Data Scientist, my professional journey is anchored in the dynamic realm of computational protein biology. My passion lies in deciphering, prevention and developing new diagnostic and therapeutic approaches through advanced data analysis. My primary focus is on the design and dynamics of proteins, as well as their interactions and interfaces. With a keen eye on detail and commitment to innovation, I am poised to continuously learn and work with new data, computational methods and technologies to expand my expertise and make a significant contribution to the drug discovery field.

## Skills
**Computational Protein interaction interface prediction:**\
3 years of experience with AlphaFold (prediction of protein structures and discovering interfaces using pLDDT and DockQ metrics) for identifying the binding protein interfaces and binding regions, MutaBind2 for designing single residue mutations and PyMOL for visualizing protein structural models

**Deep Structural Knowledge of Proteins and Peptides:**\
6 years of academic experience in Protein Biology and protein-protein/protein-ligand interactions using computations and experimental approaches (BRET assay, ITC, SPR, page display)

**Python:**\
5 years of experience in academia and industry in implementing machine learning models, data analysis and data visualization. 

**SQL:**\
3 years work in CRUD operations and Data Management using MySQL

**Project management:**\
6 years of experience in the supervised and independent implementation of projects with the final product as publication, collaboration, development of product and successful research proposal application

**Communication:**\
6 years of experience in delivering and translating concepts and progress of projects to a scientific community, potential collaborators, investors and customers

**Other software:**\
Git, Bash, Visual Studio Code, PyCharm, Microsoft 365, Miro

**Languages:**\
English (fluent), Russian (native speaker), German (A2)

## Key Publications
- Lee, C.Y., Hubrich, D., Varga,(…),O.Schueler-Furman, K.Luck. (2023). Systematic discovery of protein interaction interfaces using AlphaFold and experimental validation. BioRxiv. 10.1101/2023.08.07.552219.
- Ebersberger, S., Hipp, C., Mulorz, M.M., Buchbender, A., Hubrich, D.,(...), K.Luck, M.Sattler, J.König. (2023). FUBP1 is a general splicing factor facilitating 3’ splice site recognition and splicing of long introns. Mol. Cell 83, 2653-2672.e15. 10.1016/j.molcel.2023.07.002.
- Pascarelli, S., Merzhakupova, D., Uechi, G.-I., and Laurino, P. (2021). Binding of single-mutant epidermal growth factor (EGF) ligands alters the stability of the EGF receptor dimer and promotes growth signaling. J. Biol. Chem. 297, 100872. 10.1016/j.jbc.2021.100872.

## Experience
**Data Scientist** – 02/2023 - 10/2023\
be factory UG / Hannover\
Worked with large datasets by scraping web data, formatting, cleaning, generating new data and performing statistical analysis using python. Performed cluster analysis on skincare related data. Contributed significantly to the development of  the pilot app version - skin.sure. 

## Education
**PhD - Computational and Protein Biology** – 12/2020 - 08/2024\
Institute of Molecular Biology / Mainz, Germany\
Focus in prediction and characterization of protein interaction interfaces by designing DMI interface prediction tool (DMIPred) and implementing protein fragmentation approach using AlphaFold followed by experimental validation. Furthermore, I systematically interpreted genetic variants associated with neurodevelopmental disorders with the combination of data science and experimental approaches.

**Master of Science - Protein Design and Engineering** – 09/2016 - 05/2019\
Okinawa Institute of Science and Technology / Okinawa, Japan\
Focus in advancing our understanding of protein co-evolution using ligand-receptor EGF-EGFR system. Here, I applied the computational-based method DIRpredfor for predicting residues that are likely to be responsible for functional divergence among paralogs sharing a common interactor. 

**Cosmetology & Food Biotechnology Summer program** – 06/2017 - 07/2017\
Sup'Biotech, école de biotechnologies / Paris, France\
I participated in cosmetology product development and food engineering courses, culminating in a successful team project presentation focusing on the development and analysis of genetically modified enzymes used in bacteria-based skin and food products.

**Bachelor of Science - Molecular Biology** – 09/2011 - 06/2016\
Nazarbayev University / Astana, Kazakhstan\
Obtained general knowledge on Biological Sciences with primary focus on molecular biology and Bioinformatics.

## Certificates & Training
Hertie DS3 - Data Science – 08/2023 - 08/2023\
Motif conference – 06/2023\
Data Visualization for Scientists – 05/2022 - 06/2022\
EMBL Exploring human genetic variation – 05/2022\
Good Manufacturing Practice Course – 06/2021\
Image processing and analysis course – 03/2021\
True data - How to produce high quality data research results – 02/2021\
